src/util/syscall_sandbox.h

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_UTIL_SYSCALL_SANDBOX_H
+#define BITCOIN_UTIL_SYSCALL_SANDBOX_H
+
+enum class SyscallSandboxPolicy {
+    // 1. Initialization
+    INITIALIZATION,
+    INITIALIZATION_DNS_SEED,
+    INITIALIZATION_LOAD_BLOCKS,
+    INITIALIZATION_MAP_PORT,
+    INITIALIZATION_TOR_CONTROL,
```

Comment:
I don't think Torcontrol should be under "initialization", it's always active and keeps connecting to Tor and spinning up its onion service as long as bitcoind is running.